ci/release/update-version.sh

```diff
@@ -32,7 +32,9 @@ function sed_runner() {
 }
 
 sed_runner 's/'"CUML VERSION .* LANGUAGES"'/'"CUML VERSION ${NEXT_FULL_TAG} LANGUAGES"'/g' cpp/CMakeLists.txt
```

Comment:
We need a  sed expression for `python/CMakeLists.txt` so that we capture `set(CUML_VERSION ...)`